source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -322,6 +322,15 @@ export class HostConfig {
         large: 21,
         extraLarge: 26
     };
+
+    readonly lineHeights = {
```

Comment:
No, we can't do that. lineHeights needs to be optional to maintain backwards compat. Please add a ILineHeightDefinitions interface, then declare the lineHeights property in HostConfig as follows:

```
lineHeights?: ILineHeightDefinitions,
```

Reply:
done